nix_update/__init__.py

```diff
@@ -41,7 +41,7 @@ def parse_args() -> Options:
         "--shell", action="store_true", help="provide a shell with the package"
     )
     parser.add_argument(
-        "--version", nargs="?", help="Version to update to", default="auto"
+        "--version", nargs="?", help="Version to update to", default="stable"
```

Comment:
Mind having a look at the new code? All that it is left to do is to add this to the README.